### DISCUSSION TITLE
ContentGroupDescription

### DISCUSSION THREAD

**Discussion by milesdream** [Zeugma440/atldotnet#205] [Ideas]
I miss a Tag for ContentGroupDescription, see Microsoft WM/ContentGroupDescription
https://learn.microsoft.com/en-us/windows/win32/wmformat/wm-contentgroupdescription
regards
Robert Stefanz

**Comment by Zeugma440** [comment]
I'm pretty sure you can write it using `Track.AdditionalFields`
